doc/design/object-bucket-lib.md

```diff
@@ -138,44 +148,45 @@ This feature came late for PVCs, see [merged pr](https://github.com/kubernetes/c
 ### Bucket Sharing
 A bucket can be shared, at least within the same namespace.
 The reason for this is that the app pods never reference the OBC (or OB) directly, but instead consume a Secret and ConfigMap in order to access the bucket.
-Since more than one pod can ingest the same Secert and ConfigMap, a bucket can be shared. _TODO: verify._
+Since more than one pod can ingest the same Secert and ConfigMap, a bucket can be shared.
 
 ### Quota
+(applicable only to new buckets)
+
 Bucket size generally cannot be specified; however, the current size of a bucke can usually be monitored.
 The number of buckets can be controlled by a resource quota once [this k8s pr](https://github.com/kubernetes/kubernetes/pull/72384) is merged.
 Until then, Resource Quotas cannot yet be defined for CRDs and, thus, there is no quota on the number of buckets.
 
 ### Watches
 
 #### OBC Watches
-All provisioners importing the bucket library watch OBCs across all namespaces.
-Each binary importing the lib performs the same OBC watch.
+All provisioners importing the bucket library watch all OBCs across all namespaces.
```

Comment:
Provisioners can pass a `namespace` value to NewProvisioner() to limit (via logic, not RBAC) which namespace the provisioner will operate.  if none is given, then it assumes cluster scope